best_bank_as/models/ledger.py

```diff
@@ -8,5 +11,37 @@ class Ledger(base_model.BaseModel):
     """Model for ledger."""
 
     transaction_id = models.ForeignKey(Transaction, on_delete=models.CASCADE)
-    account_number = models.ForeignKey("best_bank_as.account", on_delete=models.CASCADE)
+    account_number = models.ForeignKey("best_bank_as.Account", on_delete=models.CASCADE)
     amount = models.DecimalField(max_digits=10, decimal_places=2)
+
+    @classmethod
+    @transaction.atomic
+    def transfer(
+        cls, source_account: Any, destination_account: Any, amount: Decimal
+    ) -> None:
```

Comment:
```suggestion
    from typing import TYPE_CHECKING
    
    # import models to be used for typing
    if TYPE_CHECKING:
    	# is only imported for type checking, and not at runtime - preventing circular imports
    	from best_bank_as.models.account import Account

    @classmethod
    @transaction.atomic
    def transfer(
        cls, source_account: Account, destination_account: Account, amount: Decimal
    ) -> None:
```

About the typing we talked about